README.md

```diff
@@ -17,16 +17,72 @@ standard implementation of lightweight Virtual Machines (VMs) that feel and
 perform like containers, but provide the workload isolation and security
 advantages of VMs.
 
+## License
+
+The code is licensed under an Apache 2.0 license.
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently runs on 64-bit systems supporting the following
+technologies:
+
+| Platform | Architecture | Virtualization technology |
+|-|-|-|
+| [Intel](https://www.intel.com) | `x86_64` | VT-x |
+| [AMD](https://www.amd.com) | `amd64` | SVM |
+| [ARM](https://www.arm.com) | `aarch64` ("`arm64`")| Hyp |
+| [IBM](https://www.ibm.com) Power | `ppc64le` |  |
+| [IBM](https://www.ibm.com) Z mainframes | `s390x` | SIE |
+
+### Hardware requirements
+
+The [Kata Containers runtime](src/runtime) provides a command to
+determine if your host system is capable of running and creating a
+Kata Container:
+
+```bash
+$ kata-runtime check
+```
+
+> **Notes:**
+>
+> - This command runs a number of checks including connecting to the
+>   network to determine if a newer release of Kata Containers is
+>   available on GitHub. If you do not wish this to check to run, add
+>   the `--no-network-checks` option.
+>
+> - By default, only a brief success / failure message is printed.
+>   If more details are needed, the `--verbose` flag can be used to display the
+>   list of all the checks performed.
+>
+> - If the command is run as the `root` user additional checks are
+>   run (including checking if another incompatible hypervisor is running).
+>   When running as `root`, network checks are automatically disabled.
+
 ## Getting started
 
 See the [installation documentation](docs/install).
 
 ## Documentation
 
-See the [official documentation](docs)
-(including [installation guides](docs/install),
-[the developer guide](docs/Developer-Guide.md),
-[design documents](docs/design) and more).
+See the [official documentation](docs) including:
+
+- [Installation guides](docs/install)
+- [Developer guide](docs/Developer-Guide.md)
+- [Design documents](docs/design)
+  - [Architecture overview](docs/design/architecture)
+
+## Configuration
+
+See the [runtime configuration details](src/runtime/README.md#configuration) and
+the [stateless systems](src/runtime/README.md#stateless-systems)
+documentation.
```

Comment:
I'm undecided whether this is the best approach (reference the config details in the runtime's README) or whether we should just move those details to the top-level README since the config file doesn't just apply to the runtime. Thoughts folks?

Reply:
The config file doesn't just apply to the runtime, but doesn't everything that we describe in that README only apply to the runtime?

Reply:
@Jakob-Naucke - I'm not totally clear on what you're saying here. The license and platform support applies to the whole project so that's why I've moved it to the top-level. The config is a bit more debatable since although it's technically for the runtime (it's read by the runtime), it also affects other parts of the system (such as the agent). Are you saying we should move that config detail into the top-level too? That would make the information more visible but I left it in the runtime README since if we moved the full details to the top-level README, it felt a bit odd for the runtime's README to refer to another file for _it's_ configuration. It's debatable though ;)

Reply:
Sorry, I'm too brief lately :) I meant: While the Kata Containers config does not only affect the runtime, AFAICT, everything we describe in https://github.com/kata-containers/kata-containers/tree/main/src/runtime#configuration does only affect the runtime. Thus, right now, I don't see a reason to move it up anyway.

Reply:
Ack. I've updated the Configuration for both README's to give a bit of extra information to users.